frontend/src/components/mine/ContactInfo/PartyRelationships/DefaultContact.js

```diff
@@ -0,0 +1,94 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import * as CustomPropTypes from "@/types";
+import { Button, Icon, Popconfirm } from "antd";
+import { GREEN_PENCIL } from "@/constants/assets";
+
+const propTypes = {
+  partyRelationship: CustomPropTypes.partyRelationship.isRequired,
+  partyRelationshipTypeLabel: PropTypes.string.isRequired,
+  handleChange: PropTypes.func.isRequired,
+  mine: PropTypes.object.isRequired,
+  openEditPartyRelationshipModal: PropTypes.func.isRequired,
+  onSubmitEditPartyRelationship: PropTypes.func.isRequired,
+  removePartyRelationship: PropTypes.func.isRequired,
+};
+
+export class DefaultContact extends Component {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
```

Comment:
This is one of the eslint rules we mentioned to look out for, As per the rule `react/destructuring-assignment` we have decided to go with 'never' for destructuring props. 

Reason being that this rule does not play nice with Redux props, so our options were to make things more complicated by choosing `always`, go with 'never', or disable the rule, 

We opted against disabling due to the gray area it will cause, and we wanted to make the frontend as consistent as possible. 

This, of course, can be discussed further if you disagree or need clarification.

Reply:
I thought that you and Nathan had decided on de-structuring as a recommendation. I'm happy to go the other way, I find de-structuring awkward. :)

Reply:
Changed.